build/copy-output.sh

```diff
@@ -19,7 +19,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```

Comment:
we could disable this lint instead, but I suspect having it on might catch actual bugs? 

these are functionally equivalent, but the updated form is more explicit, `BASH_SOURCE` should be an array.